commons-app/src/main/java/cz/cuni/mff/xrg/odcs/commons/app/dao/db/DbAuthorizatorImpl.java

```diff
@@ -23,32 +30,112 @@
  */
 class DbAuthorizatorImpl implements DbAuthorizator {
 
+    private static final String CAN_SEE_ALL = "spravca.transformacii";
+
     @Autowired(required = false)
     private AuthenticationContext authCtx;
 
+    /**
+     * Application's configuration.
+     */
+    @Autowired
+    protected AppConfig appConfig;
+
     @Override
     public Predicate getAuthorizationPredicate(CriteriaBuilder cb, Path<?> root, Class<?> entityClass) {
+        boolean ownedByOrganization = "organization".equals(appConfig.getString(ConfigProperty.OWNERSHIP_TYPE));
```

Comment:
"organization".equals(appConfig.getString(ConfigProperty.OWNERSHIP_TYPE));

Plain String must NOT be used in the comparison (it is used on more than one place), rather a public static field should be used to hold ownership.type from config.properties